tests/commands/update/test_update_panel_cmd.py

```diff
@@ -27,6 +27,30 @@ def test_update_panel(mock_app):
     )
     assert "panels.$.updated_at" in result.output
 
+    # Test adding a non-existing user
+    result = runner.invoke(
```

Comment:
I think these should be split into multiple tests as it is much harder to follow what is wrong when these starts to fail